integration-tests/src/test/java/oracle/weblogic/kubernetes/ItRemoteConsole.java

```diff
@@ -0,0 +1,196 @@
+import static oracle.weblogic.kubernetes.utils.CommonTestUtils.createSecretWithUsernamePassword;
+import static oracle.weblogic.kubernetes.utils.CommonTestUtils.dockerLoginAndPushImageToRegistry;
+import static oracle.weblogic.kubernetes.utils.CommonTestUtils.installAndVerifyWlsRemoteConsole;
+import static oracle.weblogic.kubernetes.utils.CommonTestUtils.setPodAntiAffinity;
+import static oracle.weblogic.kubernetes.utils.CommonTestUtils.shutdownWlsRemoteConsole;
+import static oracle.weblogic.kubernetes.utils.ThreadSafeLogger.getLogger;
+import static org.junit.jupiter.api.Assertions.assertNotNull;
+import static org.junit.jupiter.api.Assertions.assertTrue;
+
+@DisplayName("Test WebLogic remote console connecting to mii domain")
+@IntegrationTest
+class ItRemoteConsole {
+
+  private static String domainNamespace = null;
+
+  // domain constants
+  private static final String domainUid = "domain1";
+  private static final String clusterName = "cluster-1";
+  private static final int replicaCount = 1;
+  private static final String adminServerPodName = domainUid + "-" + ADMIN_SERVER_NAME_BASE;
+  private static final String managedServerPrefix = domainUid + "-" + MANAGED_SERVER_NAME_BASE;
+  private static LoggingFacade logger = null;
+
+  /**
+   * Get namespaces for operator and WebLogic domain.
+   *
+   * @param namespaces list of namespaces created by the IntegrationTestWatcher by the
+   *                   JUnit engine parameter resolution mechanism
+   */
+  @BeforeAll
+  public static void initAll(@Namespaces(2) List<String> namespaces) {
+    logger = getLogger();
+    // get a unique operator namespace
+    logger.info("Getting a unique namespace for operator");
+    assertNotNull(namespaces.get(0), "Namespace list is null");
+    final String opNamespace = namespaces.get(0);
+
+    // get a unique domain namespace
+    logger.info("Getting a unique namespace for WebLogic domain");
+    assertNotNull(namespaces.get(1), "Namespace list is null");
+    domainNamespace = namespaces.get(1);
+
+    /* Later we are going to verify connecting K8S WebLogic domain through remote console.
+    // install and verify operator
+    installAndVerifyOperator(opNamespace, domainNamespace);
+
+    // create a basic model in image domain
+    createAndVerifyMiiDomain();
+    */
+
+  }
+
+  /**
+   * Verify WLS Remote Console installation is successful.
+   */
+  @Test
+  @DisplayName("Verify WLS Remote Console installation is successful")
+  public void testWlsRemoteConsoleInstallation() {
+
+    assertTrue(installAndVerifyWlsRemoteConsole(), "Remote Console installation failed");
+
+  }
+
+  /**
+   * Shutdown WLS Remote Console.
+   */
+  @AfterAll
+  public void tearDownAll() {
+
+    assertTrue(shutdownWlsRemoteConsole(), "Remote Console shutdown failed");
+
+  }
+
+  /**
+   * Create a model in image domain and verify the server pods are ready.
+   */
+  private static void createAndVerifyMiiDomain() {
```

Comment:
See if you can use the utility method for createAndVerifyMiiDomain() from CommonMiiTestUtils

Reply:
I can't. Need to setup nodeport.

Reply:
Done